packages/client/components/AnalyticsPage.tsx

```diff
@@ -61,8 +62,24 @@ const AnalyticsPage = () => {
       window.analytics = mockSnippet
     }
   }, [])
-  const [isSegmentLoaded] = useScript(`https://cdn.segment.com/analytics.js/v1/${key}/analytics.min.js`, {crossOrigin: true})
+  const [isSegmentLoaded] = useScript(
+    `https://cdn.segment.com/analytics.js/v1/${key}/analytics.min.js`,
+    {
+      crossOrigin: true
+    }
+  )
   const atmosphere = useAtmosphere()
+  useEffect(() => {
+    const email = window.localStorage.getItem(LocalStorageKey.EMAIL)
+    LogRocket.init(window.__ACTION__.logRocket, {
```

Comment:
Can we follow our pattern in DataDog where you gate this initialization with another env variable, e.g., `LOGROCKET_ENABLED`? That way if this thing is harming our app performance, we can turn it off on the fly by setting the env variable, without the need of an emergency deployment?

Reply:
I like it! Just pushed an update